libs/counterfactuals/src/lib/CounterfactualList.tsx

```diff
@@ -87,6 +87,13 @@ export class CounterfactualList extends React.Component<
   public render(): React.ReactNode {
     const items = this.getItems();
     const columns = this.getColumns();
+    const targetFeature = this.getTargetFeatureName();
+    for (const column of columns) {
+      if (targetFeature !== undefined && column.fieldName === targetFeature) {
+        column.name = `${this.getTargetPrefix()} (${column.fieldName})`;
```

Comment:
it could be better to put whole string in to localization with place holder
sth like example

"Counterfactual Ex {0}"
localization.formatString(
            localization.Counterfactuals.counterfactualEx,
            i + 1
          )